src/datasources/common/json-schema.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import Ajv, { JSONSchemaType, ValidateFunction } from 'ajv';
+
+@Injectable()
+export class JsonSchemaService {
```

Comment:
What do you think about moving this to `src/common/schema` or similar?

I don't see this as a "datasource" for the routes 🤔

Reply:
I left this into the `datasources` tree because I thought this validation tools would be used just to validate external services (APIs) data. But I'm open to use this to validate client requests instead of [Nest standard validation pipes](https://docs.nestjs.com/techniques/validation#validation). In that case it makes sense to move this outside `datasources`, of course.

Reply:
Fixed in [dc0377f](https://github.com/5afe/safe-client-gateway-nest/commit/dc0377fa957d443f10ee5d31c65f3da203e40628)